File: An2/MPP/mpp-proiect-csharp-DeepSmeag/models/Inscriere.cs
using mpp_proiect_csharp_DeepSmeag.models.entity;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace mpp_proiect_csharp_DeepSmeag.models
{
    [Serializable]
    public class Inscriere : Entitate<string>
    {
        public int IdParticipant { get; }
        public int IdProba { get; }
        public Inscriere(int idParticipant, int idProba) : base(idParticipant.ToString() + "" + idProba.ToString())
        {

            this.IdParticipant = idParticipant;
            this.IdProba = idProba;
        }
    }
}

File: An2/MPP/mpp-proiect-java-DeepSmeag/modelModule/src/main/java/contests/model/Inscriere.java
package contests.model;

import contests.model.entity.Entitate;

public class Inscriere extends Entitate<String> {
    private int idParticipant;
    private int idProba;
    public Inscriere(int idParticipant, int idProba) {
        super(idParticipant + "" + idProba);
        this.idParticipant = idParticipant;
        this.idProba = idProba;
    }

    public int getIdParticipant() {
        return idParticipant;
    }

    public int getIdProba() {
        return idProba;
    }
}
